src/main/arakoon.ml

```diff
@@ -278,6 +278,8 @@ let main () =
                                       then end_i := (Some (Scanf.sscanf is "%Li" (fun i -> i))))
                                 ],
      "<tlog_dir> <tlf_dir> <path-to-db> [<end-i>]");
+    ("-load-plugin", Arg.String (fun plugin -> Dynlink.loadfile_private plugin),
```

Comment:
why not `Arg.String Dynlink.loadfile_private,` ?

Reply:
No good reason. I guess I sometimes like being a bit more verbose/explicit. I'll shorten it